orders/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db       import models
+from products.models import Product
+from users.models    import User
+
+
+class Cart(models.Model):
+    user    = models.ForeignKey(User, on_delete=models.CASCADE)
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
+    count   = models.IntegerField(default=1)
+
+    class Meta:
+        db_table = 'carts'
+
+class Order(models.Model):
+    order_number = models.IntegerField()
+    user         = models.ForeignKey(User, on_delete=models.CASCADE)
+    order_status = models.ForeignKey('Order', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'orders'
+
+class OrderStatus(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = 'order_status'
+
+class OrderItems(models.Model):
```

Comment:
Model class명은 단수로 만들어주세요!

```suggestion
class OrderItem(models.Model):
```

Reply:
단수로 변경하였습니다.